prefect_census/syncs.py

```diff
@@ -334,6 +335,7 @@ def __init__(self, census_sync, run_id) -> None:
     async def wait_for_completion(self):
         """Wait for the Census sync run to complete."""
         seconds_waited_for_run_completion = 0
+        _last_status = None
```

Comment:
Nit

Since this is not accessible by the user anyways, I don't think we need to prefix it with `_`